rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMStateCaching.java

```diff
@@ -40,60 +40,37 @@
 
 
 /**
- * Periodically request {@link RMProxyUserInterface#getMonitoring()} and store it locally.
- * <p>
- * The {@link RMInitialState} fetched from {@link RMProxyUserInterface} is a large object
- * that is long to serialize, but is always the same for every client.
- * <p>
- * Use this class to start a thread that will periodically get this object
- * using a watcher account, making the cached version available to any client instantly.
- * <p>
- * Refresh rate can be configured using {@link PortalConfiguration#RM_CACHE_REFRESHRATE}
+ * Creates new AO {@link RMProxyUserInterface} in order to access filtered events.
  */
 public class RMStateCaching {
 
     private static Logger logger = ProActiveLogger.getLogger(RMStateCaching.class);
 
     private static RMProxyUserInterface rm;
```

Comment:
Is this reference really still needed? So far I see only one usage

Reply:
rm reference to the object, aorm - to the active object. the idea is to make an object, turn it to active, but still continue use it here as normal object